src/components/Projects/TrendingProjectCard.tsx

```diff
@@ -0,0 +1,197 @@
+        .mul(100)
+        .div(volumeBeforeTrendingWindow)
+        .toNumber()
+    } else {
+      percentGain = 0
+    }
+  }
+
+  return project ? (
+    <a
+      style={{
+        borderRadius: radii.lg,
+        cursor: 'pointer',
+        overflow: 'hidden',
+      }}
+      key={project?.handle}
+      href={`/#/p/${project?.handle}`}
+    >
+      {metadata ? (
+        <div style={cardStyle} className="clickable-border">
+          <div
+            style={{ marginRight: 20, display: 'flex', alignItems: 'center' }}
+          >
+            <div style={rankStyle}>{rank}</div>
+            <ProjectLogo
+              uri={metadata.logoUri}
+              name={metadata.name}
+              size={size === 'sm' ? 70 : 110}
+            />
+          </div>
+
+          <div
+            style={{
+              flex: 1,
+              minWidth: 0,
+              fontWeight: 400,
+            }}
+          >
+            <h2
+              style={{
+                color: colors.text.primary,
+                margin: 0,
+                overflow: 'hidden',
+                textOverflow: 'ellipsis',
+                fontSize: size === 'sm' ? 16 : 21,
+              }}
+            >
+              {metadata.name}
+            </h2>
+
+            {size === 'sm' ? null : (
+              <div>
+                <span style={{ color: colors.text.primary, fontWeight: 500 }}>
+                  @{project?.handle}
+                </span>
+                <span
+                  style={{
+                    marginLeft: 10,
+                    color: colors.text.tertiary,
+                    fontSize: '0.7rem',
+                    fontWeight: 500,
+                  }}
+                >
+                  V{terminalVersion}
+                </span>
+              </div>
+            )}
+
+            <div
+              style={{
+                color: colors.text.primary,
+                display: 'flex',
+                flexWrap: 'wrap',
+                width: '100%',
+              }}
+            >
+              <span style={{ display: 'flex', flexWrap: 'wrap' }}>
+                <span style={{ fontWeight: 600 }}>
+                  <CurrencySymbol currency={CURRENCY_ETH} />
+                  {formatWad(project.trendingVolume, { precision })}{' '}
+                </span>
+                last {trendingWindowDays} days{' '}
+              </span>
+              {percentGain > 0 ? (
+                <span style={{ color: colors.text.header, fontWeight: 600 }}>
+                  +{percentGain}%
+                </span>
+              ) : null}
+            </div>
+
+            {metadata.description && (
+              <Tooltip title={metadata.description} placement="bottom">
+                <div
+                  style={{
+                    maxHeight: 20,
+                    color: colors.text.tertiary,
+                    fontWeight: 500,
+                    fontSize: size === 'sm' ? 13 : 14,
```

Comment:
**suggestion (non-blocking)**: ideally we'd be using `rem` for all CSS sizing/spacing because pixels aren't scalable: https://chiamakaikeanyi.dev/sizing-in-css-px-vs-em-vs-rem/

Reply:
Ok this is in a lot of places, so we'll get them all fixed in later PR's